spec/models/case_management/icasework/published_request_spec.rb

```diff
@@ -0,0 +1,148 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CaseManagement::Icasework::PublishedRequest, type: :model do
+  let(:request) { described_class.new(api_response) }
+
+  let(:api_response) do
+    Icasework::Case.new(attributes)
+  end
+
+  let(:attributes) do
+    {
+      case_details: {
+        case_id: '000001',
+        case_type: 'Information request',
+        case_label: 'IR - An information request about budget'
+      },
+      case_status: {
+        status: 'Some information sent but not all held'
+      },
+      case_status_receipt: {
+        method: 'Online Form',
+        time_created: '2021-02-15T16:42:19'
+      },
+      attributes: {
+        details: 'Please provide the amount spent on printers in 2020.',
+        information_will_be: 'Embedded in the response',
+        publish_in_the_disclosure_log: 'Yes',
+        reason_information_not_held: 't',
+        suitable_for_publication_scheme: 'Yes'
+      },
+      classifications: [
+        { group: 'G1', __content__: 'Bins' },
+        { group: 'G2', __content__: 'Waste' },
+        { group: 'G3', __content__: 'Missed bins' }
+      ],
+      documents: [
+        'https://example.com/?ref=D225851&access_token=VALID_ONE_HOUR'
```

Comment:
This is more like what we get:
```suggestion
        { name: 'Response', type: 'applicaion/pdf', __content__: 'https://example.com/?ref=D225851&access_token=VALID_ONE_HOUR' }
```

Reply:
Fixed in ce29eba